app/Repositories/AlertsRepository.php

```diff
@@ -23,13 +20,7 @@ class AlertsRepository
     public static function createAlertViaRequest(Request $request, Photo $photo)
     {
         $alert_type = $request->input('alert_type');
-        $alert = $alert_type === 'App\\User' ?
-            User::where('uuid', $request->input('alert_id'))->firstOrFail() :
-            (
-                $alert_type === 'App\\Pet' ?
-                    Pet::where('uuid', $request->input('alert_id'))->firstOrFail() :
-                    Device::where('uuid', $request->input('alert_id'))->firstOrFail()
-            );
+        $alert = HandleBindingRepository::bindResourceInstance($alert_type, $request->input('alert_id'));
```

Comment:
Just resolve the instance using bindResourceInstance method of HandleBindingRepository class.